Sources/Puree/Output.swift

```diff
@@ -40,4 +41,9 @@ public extension Output {
 
     func suspend() {
     }
+
+    var storageGroup: String {
+        let typeName = String(describing: type(of: self))
+        return "\(tagPattern.pattern)_\(typeName)"
```

Comment:
You're changing the behavior of existing code without any API change and that makes it easy to overlook.
What happens to the logs with the old directory name?

Making the definition of `storageGroup` the following would keep backwards compatibility.
```
var storageGroup: String {
    return tagPattern.pattern
}
```

Also what happens when the user renames a class that conforms to `Output`?